.github/workflows/build_linux.yml

```diff
@@ -0,0 +1,40 @@
+name: linux-wheels
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get tags
+      shell: bash
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
+    - uses: ./.github/workflows/actions/manylinux2010_x86_64
+    - name: copy manylinux wheels
+      run: |
+        mkdir dist
+        cp wheelhouse/cf_units*-manylinux2010_x86_64.whl dist/
+
+    - name: CheckFiles
+      shell: bash
+      run: |
+        ls dist
+    
+    - name: Test wheels
+      shell: bash
+      run: |
+        python -m pip install wheel twine pytest
+        cd dist && python -m pip install cf_units*.whl && python -m pytest --pyargs cf_units
+        python -m twine check *
+
+    - name: Publish a Python distribution to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
I created an API token on PyPI for a generic `scitools-pypi` user that we have, and also created the associated `cf-units` GitHub repository secret `PYPI_PASSWORD`